aviary/interface/methods_for_level1.py

```diff
@@ -184,6 +186,12 @@ def _setup_level1_parser(parser):
         action="store_true",
         help="Use shooting instead of collocation",
     )
+    parser.add_argument(
+        "--verbosity",
+        type=int,
+        default=1,
+        help="verbosity setting",
```

Comment:
Maybe expand this help a little bit to mention and describe the 4 verbosity levels.

Reply:
done.